mitosheet/mitosheet/code_chunks/step_performers/dataframe_steps/dataframe_delete_code_chunk.py

```diff
@@ -3,22 +3,93 @@
 
 # Copyright (c) Saga Inc.
 # Distributed under the terms of the GPL License.
-from typing import List
+from copy import copy
+from typing import List, Optional
 
 from mitosheet.code_chunks.code_chunk import CodeChunk
-
+from mitosheet.code_chunks.no_op_code_chunk import NoOpCodeChunk
 
 class DataframeDeleteCodeChunk(CodeChunk):
 
     def get_display_name(self) -> str:
         return 'Deleted Dataframe'
     
     def get_description_comment(self) -> str:
-        sheet_index = self.get_param('sheet_index')
-        df_name = self.prev_state.df_names[sheet_index]
-        return f'Deleted {df_name}'
+        sheet_indexes = self.get_param('sheet_indexes')
+        df_names = [self.prev_state.df_names[sheet_index] for sheet_index in sheet_indexes]
+        return f'Deleted {", ".join(df_names)}'
 
     def get_code(self) -> List[str]:
-        old_dataframe_name = self.get_param('old_dataframe_name')
+        old_dataframe_names = self.get_param('old_dataframe_names')
+        return [f'del {old_dataframe_name}' for old_dataframe_name in old_dataframe_names]
+
+    def _combine_right_dataframe_delete(self, other_code_chunk: "DataframeDeleteCodeChunk") -> CodeChunk:
+        first_sheet_indexes = self.get_param('sheet_indexes')
+        second_sheet_indexes = other_code_chunk.get_param('sheet_indexes')
+
+        # Because we don't have sheet ids, we need to bump any deleted dataframes
+        # that come later, so that they have the correct index now
+        for first_sheet_index in first_sheet_indexes:
+            for index, second_sheet_index in enumerate(second_sheet_indexes):
+                if first_sheet_index <= second_sheet_index:
+                    second_sheet_indexes[index] = second_sheet_index + 1
```

Comment:
Nice! Can you just clear up the comment that "come later" refers both to the sheet indexes, (first_sheet_index <= second_sheet_index), but it also refers to the step order (ie: we are only bumping the sheet indexes in the `second_sheet_indexes`, not the first.